src/components/login/ProviderLink.tsx

```diff
@@ -1,6 +1,7 @@
 import { useTranslation } from 'next-i18next'
 import { FunctionComponent, useEffect, useState } from 'react'
 import { toast } from 'react-toastify'
+import { logout } from '../../context/actions'
```

Comment:
Oh missed this tray import, don't believe it's used in this file. Probably just an IDE accident.